src/examples/algorithms/Readme.md

```diff
@@ -0,0 +1,17 @@
+## C++ NuPIC HelloWorld example
```

Comment:
nit: Why `##` for top level and `####` for second level? Using a single `#` for top level and two `##` for second level would be more consistent with usage elsewhere.